packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,34 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+
+export enum AccordionHeadingLevel {
+    h1 = "1",
+    h2 = "2",
+    h3 = "3",
+    h4 = "4",
+    h5 = "5",
+    h6 = "6",
+}
+
+export class AccordionItem extends FASTElement {
+    @attr
+    public headinglevel: AccordionHeadingLevel = AccordionHeadingLevel.h2;
+
+    @attr({ mode: "boolean" })
+    public expanded: boolean = false;
```

Comment:
Do we want this to reflect to the view if not passed by the user or do we just want the view to show `aria-expanded`?

Reply:
I tried using fromView mode but this was not changing `aria-expanded` on the accordion item. Not sure how we want to do here, the `aria-expanded` attr needs to update.

Reply:
I chatted with @EisenbergEffect sounds like we would need to do some additional work that might over complicate things to have `aria-expanded` update but not `expanded`